### PR TITLE
fixed minimap styling by props & mask-stroke-width

### PR DESCRIPTION
**Pull request by peterkogo**
- minimap styling from props now works as expected
- fixed svg styles to remove unnecessary spacing
- fixed mask-stroke-width scaling with zoom level

### CODE REVIEW

**Comment by peterkogo** [comment]
closes #3864

File: packages/svelte/src/lib/plugins/Minimap/Minimap.svelte [hunk 1]
```diff
@@ -137,6 +135,8 @@
           offset * 2}h{-viewboxWidth - offset * 2}z
       M{viewBB.x},{viewBB.y}h{viewBB.width}v{viewBB.height}h{-viewBB.width}z"
         fill-rule="evenodd"
+        stroke={maskStrokeColor}
```
Comment (moklick) [inline packages/svelte/src/lib/plugins/Minimap/Minimap.svelte:138]: couldn't we use css vars for stroke and stroke-width too?

File: packages/svelte/src/lib/plugins/Minimap/Minimap.svelte [hunk 2]
```diff
@@ -83,7 +83,7 @@
 
 <Panel
   {position}
-  {style}
+  style={style + `;--xy-minimap-background-color-props:${bgColor}`}
```
[the remark this answers is not in the document]
Comment (peterkogo) [reply packages/svelte/src/lib/plugins/Minimap/Minimap.svelte:86]: Style directives sadly don't work on svelte components, just on html tags :(

**Comment by peterkogo** [comment]
CSS variables added and made `maskStrokeWidth` 0 by default, because it was 1px & no color before which did not make much sense.

**Comment by moklick** [comment]
awesome!

**Pull request merged**